### DISCUSSION TITLE
New Adversarial Attack

### DISCUSSION THREAD

**Discussion by emancellari** [Trusted-AI/adversarial-robustness-toolbox#2402] [General]
I am Enea Mancellari from University of Luxembourg. My team and I have published a new adversarial attack, https://www.sciencedirect.com/science/article/pii/S1568494623004155. We are working in our research by using this attack. I wanted to discuss with your team if is it possible to add this attack and use it as toolbox. So in this way while publishing new articles we can refer to "Trusted-AI/adversarial-robustness-toolbox".  Could you help me with the rule/procedue of adding our attack? Best regards, Enea

**Comment by beat-buesser** [comment]
Hi @emancellari Thank you very much for considering ART for your new attack! We are looking for a diverse set of attacks, what is the most distinguishing feature of your new attack?

**Reply by emancellari** [reply]
Normally we have proceeded before with our EA attack. Our team member @aliotopal (Ali Osman Topal), he was in discussion with you and after several steps "the discussion was stopped". He did some steps and waited for your feedback. Now, I am dealing with the code and wanted to know what is the issue that we have to fix?

Thanks,

**Reply by beat-buesser** [reply]
Hi @emancellari Thank you for your update, I'm sorry, we must have missed your responses at some time among messages from other users. How does your attack distinguish itself from other black-box attack? In which scenarios is it most useful?

**Reply by emancellari** [reply]
Hi @beat-buesser, referred to our and @aliotopal discussion, we have explained in detail our attack. https://www.sciencedirect.com/science/article/pii/S1568494623004155.
Thanks,
Enea
